### PR TITLE
Create mrtkv3 button [Contnued]

### PR DESCRIPTION
**Pull request by tayomadein**
This is a continuation of the work started in this PR: https://github.com/BabylonJS/Babylon.js/pull/12707:
Assets PR: https://github.com/BabylonJS/Assets/pull/67

Preview: 

The design team is currently working on creating an interactive documentation for showcasing graphics tools components. This PR creates a TouchHolographicV3Button which mirrors the visual design of the MRTK v3.0 button.
For the time being, we would be using snapshots from this experimental PR and other PRs to create this documentation pending when we figure out the best way to release these new designs.

TODO

- [x] Figure out how to handle versioning between this button and the TouchHolographic button. As new designs for MRTK components are released, we might need to figure out the best way for users to switch from one version to the other. Otherwise, the growing list of varying components like the button might become confusing to use. - Solution: https://github.com/BabylonJS/Babylon.js/pull/12707#issuecomment-1230317075

**Playground ID - /#UHMKE6#1**

image

image

### CODE REVIEW

**Review by RaananW — COMMENTED** [review]
A few nits and a general question - 

The PR contains a lot of static values that cannot be changed - colors, sizes, scaling factors, positions and so on. Shouldn't the dev be able to change these values if required?
Same thing in the animations loop - there are many static values there that are unconfigurable

Otherwise - LGTM

File: packages/dev/gui/src/3D/controls/MRTKv3/touchHolographicButton.ts [hunk 1]
```diff
@@ -0,0 +1,1025 @@
...
+import { Vector3 } from "core/Maths/math.vector";
+
+/**
+ * Class used to create the mrtkv3 button
+ */
+export class TouchHolographicButton extends TouchButton3D {
+    /**
+     * Base Url for the frontplate model.
+     */
+    public static FRONTPLATE_MODEL_BASE_URL = "https://raw.githubusercontent.com/chinezenwosu-ms/TempAssets/main/GLB/";
```
[the remark this answers is not in the document]
Comment (tayomadein) [reply packages/dev/gui/src/3D/controls/MRTKv3/touchHolographicButton.ts:40]: @RaananW I am getting this response below for the new assets but the old assets do not throw any error:

`RuntimeError: Unable to load from https://assets.babylonjs.com/meshes/MRTK/mrtk-fluent-frontplate.glb: LoadFileError: Error status: 404  - Unable to load https://assets.babylonjs.com/meshes/MRTK/mrtk-fluent-frontplate.glb`

[the remark this answers is not in the document]
Comment (RaananW) [reply packages/dev/gui/src/3D/controls/MRTKv3/touchHolographicButton.ts:40]: You will need to add it to the assets library here - 
https://github.com/BabylonJS/Assets/tree/master/meshes/MRTK

[the remark this answers is not in the document]
Comment (tayomadein) [reply packages/dev/gui/src/3D/controls/MRTKv3/touchHolographicButton.ts:40]: @RaananW It was added in this PR https://github.com/BabylonJS/Assets/pull/67. That's why I am wondering about the 404.

File: packages/dev/gui/src/3D/controls/MRTKv3/touchHolographicButton.ts [hunk 2]
```diff
@@ -0,0 +1,1025 @@
...
+    private _frontPlateDepth = 0.2;
+    private _backPlateDepth = 0.04;
+    private _backGlowOffset = 0.1;
+    private _flatPlaneDepth = 0.001;
+    private _innerQuadRadius = this.radius - 0.04;
+    private _innerQuadColor = new Color4(0, 0, 0, 0);
+    private _innerQuadToggledColor = new Color4(0.5197843, 0.6485234, 0.9607843, 0.6);
+    private _innerQuadHoverColor = new Color4(1, 1, 1, 0.05);
+    private _innerQuadToggledHoverColor = new Color4(0.5197843, 0.6485234, 0.9607843, 1);
+
```
[the remark this answers is not in the document]
Comment (tayomadein) [reply packages/dev/gui/src/3D/controls/MRTKv3/touchHolographicButton.ts:144]: Currently there's no way to override these values. They are the default values defined for the MRTK3 buttons but I am also not sure why this portion was written this way. Looking the existing code, I think it is safe to make them changeable by the dev.

File: packages/dev/gui/src/3D/controls/MRTKv3/touchHolographicButton.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (RaananW) [reply packages/dev/gui/src/3D/controls/MRTKv3/touchHolographicButton.ts:40]: Right! deployment has failed for some reason. I am working on a fix

[the remark this answers is not in the document]
Comment (RaananW) [reply packages/dev/gui/src/3D/controls/MRTKv3/touchHolographicButton.ts:40]: Files are there, deployment fixed

**Review by RaananW — APPROVED** [review]

**Pull request merged**